src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/domain/entity/create/AuditEntity.kt

```diff
@@ -0,0 +1,58 @@
+package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.domain.entity.create
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import org.springframework.security.core.context.SecurityContextHolder
+import java.time.LocalDateTime
+import java.util.UUID
+
+@Entity
+@Table(name = "audit_record")
+class AuditEntity(
+
+  @Id
+  @Column(name = "audit_record_id")
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  var id: UUID? = null,
+
+  @Column(name = "referral_id")
```

Comment:
By the way all of these @Column annotations are unnecessary since the field names match the column name. 

I think it makes the code far more verbose than it needs to be by leaving them in but I'll leave it up to you.

And they should be val rather than var